workers/deployment_manager/ServiceAccountGenerator/Program.cs

```diff
@@ -0,0 +1,63 @@
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.ServiceAccount.V1Alpha1;
+using System;
+using System.IO;
+
+namespace ServiceAccountGenerator
+{
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length < 1)
+            {
+                Console.Out.WriteLine("Usage: <service account token path>");
+                Environment.Exit(1);
+            }
+
+            var path = args[0].Trim('"');
+            var projectName = args[1];
+
+            Console.WriteLine(Path.GetFullPath(Path.Combine(path, "ServiceAccountToken.txt")));
+
+            var perm = new Permission
+            {
+                Parts = { new RepeatedField<string> { "prj", projectName, "*" } },
```

Comment:
What does the wildcard do here?

Reply:
permission over ALL deployments in that project